modules/core/src/main/java/org/apache/ignite/cache/query/CacheQueryEntryEvent.java

```diff
@@ -45,4 +45,18 @@ public CacheQueryEntryEvent(Cache src, EventType evtType) {
      * @return Value of counter for this event.
      */
     public abstract long getPartitionUpdateCounter();
+
+    /**
+     * Cache updates on backup nodes.
+     *
+     * @return Value of remote node type for this event.
```

Comment:
I would comment this way:

Flag indicating if cache is being updated on primary node.

@retrun @code True if cache is being updated on primary node